assets/js/modules/analytics-4/datastore/settings.js

```diff
@@ -22,6 +22,11 @@
 import invariant from 'invariant';
 import { isEqual, pick } from 'lodash';
 
+/**
+ * WordPress dependencies
+ */
+import { createRegistrySelector } from '@wordpress/data';
```

Comment:
Please import from `googlesitekit-data` instead, along with everything else we use from data.